docs/content.zh/docs/get-started/quickstart/mysql-to-doris.md

```diff
@@ -104,95 +101,96 @@ Then `exit` exits and creates the Doris Docker cluster.
-1. Enter MySQL container
+该命令将以 detached 模式自动启动 Docker Compose 配置中定义的所有容器。你可以通过 docker ps 来观察上述的容器是否正常启动了，也可以通过访问[http://localhost:8030/](http://localhost:8030/) 来查看 Doris 是否运行正常。
+
+#### 在 MySQL 数据库中准备数据
+1. 进入 MySQL 容器
 
    ```shell
    docker-compose exec mysql mysql -uroot -p123456
    ```
 
-2. create `app_db` database and `orders`,`products`,`shipments` tables, then insert records
+2. 创建数据库 `app_db` 和表 `orders`,`products`,`shipments`，并插入数据
 
     ```sql
-    -- create database
+    -- 创建数据库
     CREATE DATABASE app_db;
    
     USE app_db;
    
-   -- create orders table
+   -- 创建 orders 表
    CREATE TABLE `orders` (
    `id` INT NOT NULL,
    `price` DECIMAL(10,2) NOT NULL,
    PRIMARY KEY (`id`)
    );
    
-   -- insert records
+   -- 插入数据
    INSERT INTO `orders` (`id`, `price`) VALUES (1, 4.00);
    INSERT INTO `orders` (`id`, `price`) VALUES (2, 100.00);
    
-   -- create shipments table
+   -- 创建 shipments 表
    CREATE TABLE `shipments` (
    `id` INT NOT NULL,
    `city` VARCHAR(255) NOT NULL,
    PRIMARY KEY (`id`)
    );
    
-   -- insert records
+   -- 插入数据
    INSERT INTO `shipments` (`id`, `city`) VALUES (1, 'beijing');
    INSERT INTO `shipments` (`id`, `city`) VALUES (2, 'xian');
    
-   -- create products table
+   -- 创建 products 表
    CREATE TABLE `products` (
    `id` INT NOT NULL,
    `product` VARCHAR(255) NOT NULL,
    PRIMARY KEY (`id`)
    );
    
-   -- insert records
+   -- 插入数据
    INSERT INTO `products` (`id`, `product`) VALUES (1, 'Beer');
    INSERT INTO `products` (`id`, `product`) VALUES (2, 'Cap');
    INSERT INTO `products` (`id`, `product`) VALUES (3, 'Peanut');
     ```
 
 #### Create database in Doris
-`Doris` connector currently does not support automatic database creation and needs to first create a database corresponding to the write table.
-1. Enter Doris Web UI。  
+`Doris` 暂时不支持自动创建数据库，需要先创建写入表对应的数据库。
+1. 进入 Doris Web UI。  
    [http://localhost:8030/](http://localhost:8030/)  
-   The default username is `root`, and the default password is empty.
+   默认的用户名为 `root`，默认密码为空。
 
    {{< img src="/fig/mysql-doris-tutorial/doris-ui.png" alt="Doris UI" >}}
 
-2. Create `app_db` database through Web UI.
+2. 通过 Web UI 创建 `app_db` 数据库
 
     ```sql
    create database app_db;
     ```
 
    {{< img src="/fig/mysql-doris-tutorial/doris-create-table.png" alt="Doris create table" >}}
 
-## Submit job using FlinkCDC cli
-1. Download the binary compressed packages listed below and extract them to the directory ` flink cdc-3.0.0 '`：    
-   [flink-cdc-3.0.0-bin.tar.gz](https://github.org/apache/flink/flink-cdc-connectors/releases/download/release-3.0.0/flink-cdc-3.0.0-bin.tar.gz)
-   flink-cdc-3.0.0 directory will contain four directory `bin`,`lib`,`log`,`conf`.
+## 通过 FlinkCDC cli 提交任务
+1. 下载下面列出的二进制压缩包，并解压得到目录  ` flink cdc-3.0.0 '`：    
+   [flink-cdc-3.0.0-bin.tar.gz](https://github.org/apache/flink/flink-cdc-connectors/releases/download/release-3.0.0/flink-cdc-3.0.0-bin.tar.gz).
```

Comment:
I found that this download link is invalid, can you update and fix it in English version?
The correct address is:
https://github.com/ververica/flink-cdc-connectors/releases/download/release-3.0.0/flink-cdc-3.0.0-bin.tar.gz

Reply:
Thanks, fix it.